src/components/page-types/associatesDirectoryPage/associatesDirectoryPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import SbEditable from 'storyblok-react';
+import { dcnb } from 'cnbuilder';
+import { Container } from '../../layout/Container';
+import { Heading } from '../../simple/Heading';
+import Layout from '../../partials/layout';
+import hasRichText from '../../../utilities/hasRichText';
+import RichTextRenderer from '../../../utilities/richTextRenderer';
+import CreateBloks from '../../../utilities/createBloks';
+
+const AssociatesDirectoryPage = (props) => {
+  const {
+    blok: { title, intro = {}, directory },
+    blok,
+  } = props;
+
+  return (
+    <SbEditable content={blok}>
+      <Layout hasHero={false} {...props}>
+        <Container as="main" id="main-content">
+          <header className="su-basefont-23">
+            <Container>
+              <Heading align="center" font="serif" id="page-title">
+                {title}
+              </Heading>
```

Comment:
```suggestion
              <Heading level={1} align="center" font="serif" id="page-title" className="su-max-w-900 su-mb-0 su-rs-py-5 xl:su-rs-py-7 su-type-6 su-mx-auto su-max-w-1200">
                {title}
              </Heading>
```
Let's update this to be an H1 and match the heading styles to other Alumni pages on the site. This will also take care of the top spacing!